### ISSUE TITLE
Upgrade Manifest to Version 3 for Chrome Extension

### ISSUE THREAD

**Issue by daisuke8000** [SafetyCulture/grpc-web-devtools#180]
### **Summary**

This Chrome extension has been incredibly beneficial in my daily work and has contributed significantly to improving productivity. I am deeply grateful to the developers for their efforts.

However, as Google plans to phase out Manifest Version 2 (MV2), it has become necessary to migrate this extension to **Manifest Version 3 (MV3)**.

---

### **Background**

Here are the reasons for raising this issue:

1. **Warning Message**  
   The Chrome Web Store displays a warning indicating that the extension does not align with the "best practices for Chrome extensions" because it is still using Manifest Version 2.  
   - There is a risk that this extension may soon cease to be supported by Google.

2. **MV2 Deprecation Timeline**  
   Google has officially announced the deprecation of Manifest Version 2, and starting in 2024, support for extensions using MV2 will be discontinued.  
   - Details can be found in the [official documentation](https://developer.chrome.com/docs/extensions/migrating/).

3. **Impact on the Project**  
   If this extension is not updated to Manifest Version 3, it may cease to function properly in the near future.

---

### **Proposed Solution**

To address this issue, I propose the following actions to migrate to Manifest Version 3 (MV3):

1. **Update Manifest File**  
   - Update `manifest.json` to follow the MV3 format.  
   - Adjust necessary APIs and permissions to be compatible with MV3.

2. **Address MV3-Specific Changes**  
   - Adapt to the use of service workers and address API removals or deprecations introduced in MV3.

---

### **Reference Links**

- [Chrome Official Guide: Migrating from Manifest V2 to V3](https://developer.chrome.com/docs/extensions/migrating/)

---

### **Additional Notes**

This extension plays an important role in professional workflows. Migrating to Manifest Version 3 will ensure its continued usability and compliance with Chrome Web Store requirements. I kindly request the developers to consider this migration, and I truly appreciate your continuous efforts to improve this amazing tool.

**Comment by ilias-t** [comment]
is this why it's no longer available in the chrome web store? https://chromewebstore.google.com/detail/grpc-web-developer-tools/kanmilmfkjnoladbbamlclhccicldjaj

**Comment by dimo414** [comment]
@rogchap @matt-lewandowski @georgechieng-sc is this project still maintained? It has been removed from the Google Chrome Web Store. It would be great if you could update the manifest.

It looks like @SergioTx is [maintaining a fork](https://github.com/SergioTx/grpc-web-devtools) which has a [corrected manifest](https://github.com/SafetyCulture/grpc-web-devtools/commit/184eba9ff06f517486d533be3bee77f486070acb).

**Comment by SergioTx** [comment]
Yes, I created a fork to fix a bug that was bothering me and to upload it to Chrome I updated the manifest. I also added a few small features, but I removed others that were not useful for us. Feel free to use it, send any PRs or use my code to update this one.
